internal/zinc-persist/src/main/scala/sbt/internal/inc/FileBasedStore.scala

```diff
@@ -42,8 +49,10 @@ object FileBasedStore {
             outputStream.closeEntry()
           }
       }
+      IO.move(tmpAnalysisFile, file)
```

Comment:
Good point, I'll follow up with a new PR, though I'm not sure if there's an equivalent for `REPLACE_EXISTING` in JKD 6.